src/react/account/__tests__/AccountUserList.test.tsx

```diff
@@ -124,5 +140,88 @@ describe('AccountUserList', () => {
     expect(screen.getByText(SAMPLE_USER_NAME)).toBeInTheDocument();
 
     //TODO test copy arn
+    const firstRow = screen.getAllByRole('row')[1];
+
+    const arnButtonOfFirstRow = getAllByRole(firstRow, 'button');
+    expect(getByText(
+      arnButtonOfFirstRow[1],
+      /Copy ARN/i,
+    )).toBeInTheDocument();
+
+    fireEvent.mouseEnter(screen.getAllByRole('button')[4]);
+
+    expect(
+      screen.getByText('Copy ARN'),
+    ).toBeInTheDocument();
```

Comment:
I think this is likely not testing a specific use case. IIUC here you are checking that when mouse enter the 4th button of the page an element is present somewhere in the screen with the "Copy ARN" text, which is already tested just above. What do you wanted to test here ? A tooltip ? On which button ? Could we localise the button in a better way ? Also if it is a test concerning a tooltip we should localize it more precisely than with "Copy ARN" on the global screen as we saw that just above there is another component with the same text that would match.